### DISCUSSION TITLE
Is reaching out to plugin authors the only way to get answers to questions about a plugin?

### DISCUSSION THREAD

**Discussion by Schaefer6** [runelite/runelite#17122] [Q&A] [answered]
If the question has not been answered before somewhere online? If I have a question about a given plugin, is my only option to ask the author directly? I've not had any luck with doing this and asking questions about plugins in the RuneLite discord is not permitted.

**Comment by Nightfirecat** [comment, marked as answer]
You can ask about hub plugins in Discord but you're unlikely to get an answer in #support, as that channel is specifically for support with the features we actually maintain. (that is, the core client) Often, hub plugin authors are in the server, so you can try to use the RuneLite discord bot's `/whois` command to find their handle and ask them directly there. You can also find the email for their GitHub account and email them directly.
